fedimint-cli/src/client.rs

```diff
@@ -226,13 +228,13 @@ pub async fn handle_ng_command(
                                 })
                                 .unwrap());
                             }
-                            InternalPayState::RefundSuccess(outpoint) => {
+                            InternalPayState::RefundSuccess { outpoint, error } => {
                                 let e = format!(
-                                    "Internal payment failed. A refund was issued to {outpoint}"
+                                    "Internal payment failed. A refund was issued to {outpoint} Error: {error}"
```

Comment:
I was told that by using `{x:?}` a stacktrace will not be lost.

Reply:
I actually don't think we want a stack trace here. The error in this case occurred on the gateway side, a stack trace of the CLI won't be that useful